### PR TITLE
Remove linking references

### LINKED ISSUE #3542: Remove linking references from the Console
#### Summary

References #3263

We should remove any linking references from the Console in the `3.11` branch.

#### Why do we need this?

Linking has been completely removed as of #3263, both from the AS and from the NS. Fundamentally the only thing that is still part of the `ttnpb.ApplicationLink` is the default payload formatter and the application-wide skip-payload-encryption option, but any linking options such as NS address or API key are now gone.

#### What is already there? What do you see now?

1. Automatic link creation on application creation.
2. Application link status page

#### What is missing? What do you want to see?

Both should be removed.

#### Environment

`3.11`

#### How do you propose to implement this?

I suppose it's just deleting the forms / logic. One thing to look into is if updates to the application payload formatter still work (not sure if they are currently dependent on the existence of the link ?)

#### Can you do this yourself and submit a Pull Request?

I can review this.

### PR DESCRIPTION
**Pull request by asmulko**
#### Summary

Closes #3542 

#### Changes

- Removed linking from create application 
- Removed linking from side menu
- Removed linking info from application overview
- Updated uplink/downlink not to depend on linking

Before:
before_1
After: 
after_1

Before:
before_2
After:
after_2

Before:
before_3
After:
after_3

#### Testing

Manually tested

#### Notes for Reviewers

- I would appreciate if someone could test this locally to see if nothing is broken, I could've missed something

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/application-title-section/application-title-section.js [hunk 1]
```diff
@@ -25,8 +25,6 @@ import EntityTitleSection from '@console/components/entity-title-section'
 import sharedMessages from '@ttn-lw/lib/shared-messages'
 import PropTypes from '@ttn-lw/lib/prop-types'
 
-import ApplicationStatus from './status'
```
Comment (bafonins) [inline pkg/webui/console/containers/application-title-section/application-title-section.js:28]: You can then also remove the `status` folder. Or can we still fetch up/down counts and last received timestamp from `ApplicationLinkStats` ? cc @adriansmares

File: pkg/webui/console/containers/application-title-section/application-title-section.js [hunk 2]
```diff
@@ -110,22 +101,15 @@ ApplicationTitleSection.propTypes = {
   devicesErrored: PropTypes.bool.isRequired,
   devicesTotalCount: PropTypes.number,
   fetching: PropTypes.bool.isRequired,
-  lastSeen: PropTypes.string,
-  linkStats: PropTypes.applicationLinkStats,
```
Comment (bafonins) [inline pkg/webui/console/containers/application-title-section/application-title-section.js:114]: You can also edit the `applicationLink` and `applicationLinkStats` prop-type definitions to match the current types.

Comment (bafonins) [reply pkg/webui/console/containers/application-title-section/application-title-section.js:114]: `ApplicationLink` does not require `network_server_address` and `api_key` as those will be deprecated in v3.11

**Review by bafonins — CHANGES_REQUESTED** [review]

File: pkg/webui/console/containers/application-title-section/application-title-section.js (hunk 1 above)
Comment (adriansmares) [reply pkg/webui/console/containers/application-title-section/application-title-section.js:28]: `ApplicationLinkStats` is deprecated. There are no plans in bringing it back at the moment.

**Review by adriansmares — COMMENTED** [review]
I've tested it locally and works as expected. Test procedure:

- Create application
- Create end device
- Set application uplink payload formatters to the default JS formatter
- Send simulated uplink and check uplink contents (contains correct `decoded_payload`)

**Review by bafonins — APPROVED** [review]

**Pull request merged**